### PR TITLE
Read default CR

### PR DESCRIPTION
**Pull request by pbochynski**
**Description**

Changes proposed in this pull request:

- handle properly default CR

Solves: https://github.com/kyma-project/cli/issues/1491

### CODE REVIEW

**Comment by pbochynski** [comment]
Can someone review this PR?

File: cmd/kyma/alpha/create/module/module.go [hunk 1]
```diff
@@ -195,6 +195,15 @@ func (cmd *command) Run(args []string) error {
 
 func (cmd *command) validateDefaultCR(modDef *module.Definition, l *zap.SugaredLogger) error {
 	cmd.NewStep("Validating Default CR")
+
+	if modDef.DefaultCRPath != "" {
```
Comment (ruanxin) [inline cmd/kyma/alpha/create/module/module.go:199]: This prefill DefaultCR is also done in `inspectProject` project step https://github.com/kyma-project/cli/blob/main/pkg/module/resources.go#L213-L226 , but have the precondition `if def.RegistryURL != "" `, I don't have too much detail context for this decision. IMO, if this `DefaultCR` in anyway needs to be filled when `DefaultCRPath` exits, then it's better remove that precondition in `module.Inspect`, this step is happen before `validateDefaultCR`

**Review by ruanxin — CHANGES_REQUESTED** [review]
only have some concern for the place to do this `DefaultCR` initialization.

**Comment by pbochynski** [comment]
@ruanxin I moved DefaultCR initialization before validation and removed duplicated code from another place.

**Review by ruanxin — APPROVED** [review]

**Pull request merged**